IdentityCore/src/webview/embeddedWebview/challangeHandlers/ios/MSIDCertAuthHandler.m

```diff
@@ -150,14 +150,14 @@ + (BOOL)handleChallenge:(NSURLAuthenticationChallenge *)challenge
             if ([item.name isEqualToString:MSID_OAUTH2_REDIRECT_URI]
                 && ![item.value.lowercaseString hasPrefix:s_redirectScheme.lowercaseString])
             {
-                newQueryItems[MSID_OAUTH2_REDIRECT_URI] = [s_redirectPrefix stringByAppendingString:item.value.msidWWWFormURLEncode];
+                newQueryItems[MSID_OAUTH2_REDIRECT_URI] = [s_redirectPrefix stringByAppendingString:item.value];
```

Comment:
this is a change from previous implementation, where we passed encoded redirect uri. Adding comment just to double check that it won't cause issues.

Reply:
Prev. was a bug

Reply:
We should reuse the value as they come.